src/irmin-pack/unix/dispatcher.ml

```diff
@@ -240,7 +240,7 @@ module Make (Fm : File_manager.S with module Io = Io.Unix) :
       let accessor =
         Accessor.v_exn t ~off:entry_offset_suffix_start ~len:read_in_suffix
       in
-      read_exn t accessor buf;
+      read_exn t accessor buf_suffix;
```

Comment:
with this PR, the function `read_in_prefix_and_suffix_exn` is now tested by for instance https://github.com/mirage/irmin/blob/main/test/irmin-pack/test_gc.ml#L233. 
This happens while creating `prefix.2` we are transferring a chunk of data that is split between `prefix.1` and `suffix.1`. 
(I'm adding a note here so that we can tick of the corresponding item in https://github.com/mirage/irmin/issues/2096)

Reply:
Ooh I see it now, thank you :)